app/assets/stylesheets/application_split2.css

```diff
@@ -0,0 +1,3 @@
+/*
```

Comment:
Probably don't need to do the IE9 split fix. Making a guess that Mapbox will have issues with IE9 anyway.